### PR TITLE
Checking fwatlasbc

### PR DESCRIPTION
**Pull request by dunkenwg**

### CODE REVIEW

**Comment by dunkenwg** [comment]
@joethorley, the actions are failing because four tests are failing:

```
Failure (test-add-collection-to-polygon.R:54:3): fwa_add_collection_to_polygon function intersects work
x$geometry[[1]] inherits from 'XY'/'POINT'/'sfg' not 'LINESTRING'.

Failure (test-add-collection-to-polygon.R:55:3): fwa_add_collection_to_polygon function intersects work
colnames(sf::st_coordinates(x$geometry[[1]])) (`actual`) not identical to c("X", "Y", "L1") (`expected`).

`actual`:   "X" "Y"     
`expected`: "X" "Y" "L1"

Failure (test-snap-rm-to-rms.R:412:3): fwa_snap_rm_to_rms interpolates blocks
x$new_rm (`actual`) not equal to c(1, 2, 3, 4, 5, 6, 7, 8, 10, 11, 12, 13, 13, 14, 15) (`expected`).

  `actual`: 1 2 2 3 4 6 6 7  9 10 and 5 more...
`expected`: 1 2 3 4 5 6 7 8 10 11           ...

Failure (test-snap-rm-to-rms.R:413:3): fwa_snap_rm_to_rms interpolates blocks
x$distance_to_new_rm (`actual`) not equal to c(...) (`expected`).

  `actual`: 0 0 0 874 0 0 932 0   0 0 and 5 more...
`expected`: 0 0 0   0 0 0   0 0 536 0           ...
```

We're also getting 46 instances of the same warning:

```
fwa_snap_rms_to_rms multiple blks to 1 blk partial argument match of 'rm' to 'rms'
```

Would we be able to discuss the functions and tests of this package so that I have a better idea of how to fix the tests?

File: DESCRIPTION [hunk 1]
```diff
@@ -43,4 +43,5 @@ Encoding: UTF-8
 Language: en-US
 LazyData: true
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.3.1
+RoxygenNote: 7.3.2
+URL: https://poissonconsulting.github.io/fwatlasbc/
```
Comment (joethorley) [inline DESCRIPTION:47]: This is in the wrong place - please run `usethis::use_tidy_description()`

File: .github/workflows/R-CMD-check.yaml [hunk 2]
```diff
@@ -6,7 +6,9 @@ on:
   pull_request:
     branches: [main, master]
 
-name: R-CMD-check
+name: R-CMD-check.yaml
```
Comment (joethorley) [inline .github/workflows/R-CMD-check.yaml:9]:
```suggestion
name: R-CMD-check
```

**Review by joethorley — CHANGES_REQUESTED** [review]
This seems good but going forward styler changes should be done as a separate PR to other changes - its almost impossible to manually check 75 files to see what is simply a change in style and what is a functional change.

See a couple of suggested commits.

**Comment by dunkenwg** [comment]
@joethorley, would doing styler changes after having the rest of the changes reviewed work?

**Comment by joethorley** [comment]
@dunkenwg - yes I think you should first do a PR that only fixes the breaking tests and once are happy with that we can make the other changes.

**Pull request closed without merging**